gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/io/graphml/GraphMLWriter.java

```diff
@@ -18,6 +18,21 @@
  */
 package org.apache.tinkerpop.gremlin.structure.io.graphml;
 
+import java.io.IOException;
```

Comment:
i just happened to peek back in on this PR and noticed that it looks like your IDE re-organized the imports a bit. when you come back to this could you please revert that portion of the change?